custom_components/unifiprotect/config_flow.py

```diff
@@ -34,6 +34,8 @@
 class UnifiProtectFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a Unifi Protect config flow."""
 
+    entry: Optional[config_entries.ConfigEntry] = None
```

Comment:
Usually we want to define these in `__init__` to make it clear its not constant between multiple config flows